integration-test/request-blocking.spec.js

```diff
@@ -20,14 +21,14 @@ async function runRequestBlockingTest (page) {
         }
         pageRequests.push({
             url: req.url(),
-            mathod: req.method(),
```

Comment:
Has this always been broken?

Reply:
Yep. I'm not sure we're checking this in the tests